src/app_charts/prometheus/prometheus-robot.values.yaml

```diff
@@ -61,7 +61,7 @@ prometheus:
     # Historical data is limited to the lower of `retention` and
     # `retentionSize`.
     retention: "1d"
-    retentionSize: 512MB
+    retentionSize: 1024MB
```

Comment:
Unless I'm missing something, increasing retentionSize without increasing sizeLimit below will make the problem worse, not better. I'd suggest either reducing this to "384MB" (to have some headroom to before getting disk space errors at 512MiB) and reducing retention to "3h" - or if it's unavoidable, setting sizeLimit to 1024MB and setting retentionSize to something a bit less than that.